.eslintrc.yml.jinja

```diff
@@ -1 +1 @@
-{%- include "vendor/oca-addons-repo-template/src/{% if odoo_version > 12 %}.eslintrc.yml{% endif %}" -%}
+{%- include "vendor/oca-addons-repo-template/src/{% if odoo_version > 12 %}.eslintrc.yml{% endif %}.jinja" -%}
```

Comment:
```suggestion
{%- include "vendor/oca-addons-repo-template/src/{% if odoo_version > 12 %}.eslintrc.yml.jinja{% endif %}" -%}
```

Reply:
The file name is actually `vendor/oca-addons-repo-template/src/{% if odoo_version > 12 %}.eslintrc.yml{% endif %}.jinja`
https://github.com/OCA/oca-addons-repo-template/blob/master/src/%7B%25%20if%20odoo_version%20%3E%2012%20%25%7D.eslintrc.yml%7B%25%20endif%20%25%7D.jinja

Reply:
Ah, sorry, I misunderstood it! :facepalm: